SharpBucket/V2/EndPoints/RepositoriesEndPoint.cs

```diff
@@ -21,6 +21,11 @@ public RepositoriesEndPoint(SharpBucketV2 sharpBucketV2)
         {
         }
 
+        public RepositoriesEndPoint(ISharpBucketV2 sharpBucketV2)
+            : base(sharpBucketV2, "repositories/")
```

Comment:
To avoid to duplicate the "repositories/" literal constant it would be great that the legacy constructor call this(sharpBucketV2) instead of calling the base